src/AppBundle/Resources/views/themes/app/menu.html.twig

```diff
@@ -1,3 +1,51 @@
 {% extends '@KnpMenu/menu.html.twig' %}
 
 {# To override this template, copy and paste relevant blocks from '@KnpMenu/menu.html.twig' and 'knp_menu.html.twig' #}
+
+{% block item %}
```

Comment:
What exactly is changed in this block in regards to original ?

Can you just modify some variables and call the parent, in order not to copy paste the entire thing?

Reply:
I agree. Due to the way the logic of the blocks is implemented, I cannot make a less invasive change other than copying the entire block, I believe. But the changes are actually minor and we should document them, I agree.

Reply:
@emodric I have wrapped all modified pieces of code in the comment with the Jira task number. Is that enough?

Reply:
Yep, looks good! :+1: